### PR TITLE
Update remote_docker version in CircleCI config

### PR DESCRIPTION
**Pull request by dschuff**
20.10.17 is the current default.

### CODE REVIEW

**Review by sbc100 — COMMENTED** [review]
lgtm if tests pass... should we just remove the version?

**Review by sbc100 — APPROVED** [review]

**Comment by dschuff** [comment]
Yeah, maybe just removing the version is better...

**Comment by forderud** [comment]
I've now rebased the PR on top of main after #1117 was merged.

**Pull request merged**
